scripts/Muon/GUI/Common/plot_widget/plot_widget_presenter.py

```diff
@@ -359,6 +360,10 @@ def plot_all_selected_data(self, autoscale, hold_on):
         workspace_list, indices = self._model.get_workspace_list_and_indices_to_plot(self._view.is_raw_plot(),
                                                                                      self._view.get_plot_type())
 
+        if workspace_list:
+            self._view.setEnabled(True)
+        else:
+            self._view.setEnabled(False)
```

Comment:
Is this worth adding a test for (especially as we have the mocking ready anyway) so this doesn't regress in the future?